bee-ledger/src/workers/snapshot/download.rs

```diff
@@ -175,26 +205,3 @@ pub(crate) async fn download_latest_snapshot_files(
     Err(Error::NoDownloadSourceAvailable)
 }
 
-async fn download_snapshot_file(path: &Path, download_url: &str) -> Result<(), Error> {
```

Comment:
I'm happy you did this, I wanted to ask for it but thought I would be way too pedantic

Reply:
😊